deep_reference_parser/io/io.py

```diff
@@ -98,7 +98,7 @@ def load_tsv(filepath, split_char="\t"):
         filepath.
 
     """
-    df = pd.read_csv(filepath, delimiter=split_char, header=None, skip_blank_lines=False)
+    df = pd.read_csv(filepath, delimiter=split_char, header=None, skip_blank_lines=False, encoding="utf-8", quoting=csv.QUOTE_NONE, engine="python")
```

Comment:
I didn't replicate the issue, so this is just an idea, but could issue #26 be to do with this 'quoting' parameter? like does it replace the quotes with nothing and then mess up the number of characters? Just looking at this (closed) issue https://github.com/pandas-dev/pandas/issues/23717

Reply:
hmm i don't think so, because the problem only occurs when reloading the saved weights to make predictions. If the `quoting` was an issue, I would expect to see some problems arising when we load in the data for training too.